README.md

```diff
@@ -35,6 +33,12 @@ Note: Just a heads up, the Swift Package Manager approach isn't ready for use ju
     - `pod 'Transcend'`
 - Run `pod install` to fetch the Transcend Framework.
 
+### Using Swift Package Manager
+
+```
+    https://github.com/transcend-io/Transcend-spm-sdk.git
+```
+
```

Comment:
not super familiar with the swift package manager. are there any other instructions on how to use it in the mentioned url?

Reply:
They just specify the git url on their` Package.swift` file which would download the resources from https://github.com/transcend-io/Transcend-spm-sdk.git.

Reply:
Thanks for explaining! Maybe add a sentence like that above the url just in case?